walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,7 @@
    * Returns a greeting for the given name.
```

Comment:
nit: I'd update the Java doc to include a comment that says it strips whitespace and non A-Z0-9 chars